lib/elf_py.c

```diff
@@ -1307,7 +1307,7 @@ static PyObject *elffile_load(PyTypeObject *type, PyObject *args,
 	}
 #endif
 
-	w->sects = calloc(sizeof(PyObject *), w->ehdr->e_shnum);
+	w->sects = calloc(w->ehdr->e_shnum, sizeof(PyObject *));
```

Comment:
How did this code ever work? or was it never used?

Reply:
calloc always multiples one number by the other.  This is simple multiplication rules.  The compiler is just asking for the number of elements * the size to be in a specific order

Reply:
I suppose.  :)